### ISSUE TITLE
Reverse MultipleCursorsAddJumpNextMatch

### ISSUE THREAD

**Issue by FbN** [brenton-leighton/multiple-cursors.nvim#79]
Please add a command that reverse MultipleCursorsAddJumpNextMatch

**Comment by brenton-leighton** [comment]
@FbN this is implemented in #84, let me know if there's any issue.

**Comment by FbN** [comment]
Nice addon @brenton-leighton thank you, but my request was a bit differrent I think: remove current cursor and jump to previous match. So to undo the last MultipleCursorsAddJumpNextMatch. Maybe can be done with a combination of commands?

**Comment by brenton-leighton** [comment]
Yes if you use `MultipleCursorsJumpPrevMatch` it will jump backwards without leaving a cursor behind. When the real cursor jumps it also removes any virtual cursor it lands on.

**Comment by FbN** [comment]
oh I see.. that's perfect thank you very much 😄

Il giorno lun 10 feb 2025 alle ore 01:04 Brenton Leighton <
***@***.***> ha scritto:
